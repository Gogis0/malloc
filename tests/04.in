alloc 207
free 126
free 1960
free 1618
free 1899
alloc 89
free 1313
free 1313
free 1516
free 531
free 1695
alloc 113
free 917
free 1141
free 1402
alloc -84
free 1554
alloc 143
alloc 147
alloc 213
end
