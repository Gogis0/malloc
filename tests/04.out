20
-1
-1
-1
-1
247
-1
-1
-1
-1
-1
356
-1
-1
-1
-1
-1
489
652
819
